### DISCUSSION TITLE
I need help with a problem

### DISCUSSION THREAD

**Discussion by yanzhangshuai** [vuejs/pinia#864] [Help and Questions] [answered]
I need help with a problem. That's not  pinia bug. But I'm asking for your help

![image](https://user-images.githubusercontent.com/31086862/144551868-ca2baab1-b509-4309-b248-e277e7714805.png)

But I have registered  @vue/composition-api

![image](https://user-images.githubusercontent.com/31086862/144551999-e25aa375-163f-4a4a-9a6e-437532736484.png)
![image](https://user-images.githubusercontent.com/31086862/144552011-dd8cb2a6-0c07-40ad-98e8-ce72c41c2f1f.png)

vue@2.6.14
pinia@2.0.5
@vue/composition-api@1.4.1

**Comment by BenShelton** [comment, marked as answer]
This is a quirk of the composition API in Vue2. See the Troubleshooting section in the following release for fixes: https://github.com/vuejs/composition-api/releases/tag/v1.3.0

We had a similar problem when one lib used an older version of vue-demi, so when we upgraded to Pinia@2 there were constant conflicts in resolution. If you're in a similar situation and using Vite then the following lines in your config under `resolve.alias` should fix it (as noted in the above link):

```ts
alias: [
  { find: /^@vue\/composition-api$/, replacement: '@vue/composition-api/dist/vue-composition-api.mjs' },
  { find: '@vue/composition-api/dist/vue-composition-api.esm.js', replacement: '@vue/composition-api/dist/vue-composition-api.mjs' },
  // your other aliases here
]
```
